scripts/renormalize.py

```diff
@@ -0,0 +1,39 @@
+import datetime as dt
+
+from cqlengine import Token
+
+from scrapi.database import _manager
+from scrapi.processing.cassandra import DocumentModel
+from scrapi.tasks import normalize, process_normalized
+from scrapi.linter import RawDocument
+
+_manager.setup()
+
+
+def document_generator():
+    count = 0
+    query = DocumentModel.objects.all().limit(1000)
+    page = list(query)
+    while len(page) > 0:
+        for doc in page:
+            if not isinstance(doc.providerUpdatedDateTime, dt.datetime):
+                count += 1
+                try:
+                    yield RawDocument({
+                        'doc': doc.doc,
+                        'docID': doc.docID,
+                        'source': doc.source,
+                        'filetype': doc.filetype,
+                        'timestamps': doc.timestamps
+                    })
+                except Exception as e:
+                    print(e)
+        page = list(query.filter(pk__token__gt=Token(page[-1].pk)))
+    print(count)
```

Comment:
Just wanted to make sure it iterated over the correct number of documents. Can be deleted.